eva_cttv_pipeline/evidence_strings.py

```diff
@@ -61,7 +61,7 @@ def __init__(self, a_dictionary, ensembl_gene_id=None, clinvar_record=None, ense
             try:
                 self.set_target(ensembl_gene_id_uri, clin_sig_2_activity[clin_sig])
             except KeyError:
-                # unrecognised_clin_sigs.add(clin_sig)  # TODO fix this
+                report.unrecognised_clin_sigs.add(clin_sig)  # TODO fix this
```

Comment:
Should this exception be raised instead of letting the object be fully constructed?